config/routes.rb

```diff
@@ -82,6 +82,7 @@
   resources :reviews, only: [:show, :create, :destroy, :update, :edit]
   resources :groups, only: [:create, :destroy], path: 'group_members'
   resources :quotes
+  resources :quote_anonymous, only: [:update]
```

Comment:
Dan word de route hier: 
```suggestion
resources :quotes do 
  member do 
    resource anonymization, only: [:update]
  end
end
```